docs/release-notes/3561-customize-experiment-columns.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**New Features**
+
+-  Web: Adds a column picker to the Experiment List page to allow users to choose which table
+   columns to display.
```

Comment:
**suggestion:** add some screens and/or screencast to demo this awesome new feature!